### PR TITLE
added multiexplorer wallet link

### PR DESCRIPTION
**Pull request by priestc**

### CODE REVIEW

**Comment by luke-jr** [comment]
@prusnak @slush0

**Comment by priestc** [comment]
Whats the hold up? Merge this already!

**Comment by luke-jr** [comment]
Neither author has ACK'd it. Email them directly if you like, but don't nag please.

File: bip-0044.mediawiki [hunk 1]
```diff
@@ -267,7 +267,10 @@ is required and a pull request to the above file should be created.
 * [[https://play.google.com/store/apps/details?id=com.mycelium.wallet|Mycelium Bitcoin Wallet (Android)]] ([[https://github.com/mycelium-com/wallet|source]])
 * [[https://copay.io/|Copay]] ([[https://github.com/bitpay/copay|source]])
 * [[https://maza.club/encompass|Encompass]] ([[https://github.com/mazaclub/encompass|source]])
-* [[https://www.coinvault.io/|CoinVault]] 
+* [[https://www.crypzo.com/|Crypzo]]
```
Comment (bgok) [inline bip-0044.mediawiki]: This link is broken. Connection refused.

**Comment by prusnak** [comment]
Nack. The code is not very useful to serve as an example for other developers.

**Pull request closed without merging**